modules/audit_config/variables.tf

```
@@ -0,0 +1,16 @@
+variable "service" {
+  description = "Service which will be enabled for audit logging" 
+  type        = string
+  default     = "allServices"
+}
+
+variable "audit_log_config" {
+  description = "Map of log type and exempted members to be addded to service"
+  type        = list(map(string))
```

Could this be a list of objects?

```suggestion
  type        = list(object({log_type: string, exempted_members: list(string)))
```